### ISSUE TITLE
Missing dist/index.js

### ISSUE THREAD

**Issue by luluiscute7** [etienne-martin/device-detector-js#104]
Version 2.2.3 , 2.2.4
Could not found to dist/index.js,,

so my app is crash down..!!
please fix it!

**Comment by etienne-martin** [comment]
Thanks for letting me know! Somehow adding the `resolveJsonModule` option in my previous PR completely messed up the folder structure of the `dist` folder.

I added a new test to prevent this kind of error from happening in the future: https://github.com/etienne-martin/device-detector-js/blob/master/src/tests/lib.test.ts

The issue has been fixed, version `2.2.5` is being published on npm at the moment.
